server/presenters/unfurls/unfurl.ts

```diff
@@ -0,0 +1,19 @@
+import { IframelyErrorResponse, Unfurl } from "@shared/types";
+
+function presentUnfurl(data: any): Unfurl | IframelyErrorResponse {
+  return !data.error
```

Comment:
I don't like this overloading with the error response, but it won't be needed with the other suggestion